### DISCUSSION TITLE
2024-06-22(토)

### DISCUSSION THREAD

**Discussion by Seoin02** [Together-3team/petFrontend#223] [데일리스크럼]
아래 내용을 복사해 작성해주세요.
```
### 완료한 일

### 해야할 일

### 어려운 일
```

**Comment by Seoin02** [comment]
### 완료한 일
- 프로필 이미지 삽입
- 내 리뷰 페이지 API 연동

### 해야할 일
- my order 페이지 API 연동

### 어려운 일
- 시간이 없어효... 부족해여...

**Comment by q45402sk** [comment]
### 완료한 일
상품상세 api
deliveryAddPage
deliveryEdiitPage api

### 해야할 일
옵션 선택 바텀시트

### 어려운 일

**Comment by devwqc** [comment]
### 완료한 일
- 홈 페이지
- 인기상품 페이지
- 추천상품 페이지
- 카테고리 페이지
- 찜 페이지

### 해야할 일
- 검색 페이지

### 어려운 일
- 파이팅팅팅팅팅!!! 💪
